.travis.yml

```diff
@@ -0,0 +1,30 @@
+sudo: false
+language: node_js
+addons:
+  apt:
+    packages:
+    - zsh
+bundler_args: --without production development --deployment --jobs=3 --retry=3
+cache:
+  bundler: true
+  apt: true
+  directories:
+    - coverage
+    - node_modules
+    - node_modules
+    - $(npm config get prefix)/bin/bower
+    - $(npm config get prefix)/bin/typings
+    - $(npm config get prefix)/bin/tslint
+    - $(npm config get prefix)/bin/gulp
+before_install:
+  - export TZ=Europe/Zurich
+before_script:
+  - export DISPLAY=:99.0
+  - sh -e /etc/init.d/xvfb start
+  - echo {\"token\":\"$TSDRC_TOKEN\"} > .tsdrc
+  - bin/setup
+script: bin/check
```

Comment:
can we use bin/setup instead of bin/check here? bin/setup calls bin/check in the end, right?